content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -58,6 +58,13 @@ mountOptions:
 volumeBindingMode: Immediate
 ```
 
+### Default Storage Class
+
+Typically, each Kubernetes cluster should have a default storage class defined, which would be used by the user who make a storage request without implicitly specifying a StorageClass in his/her PersistentVolumeClaim.  This is useful for Administrators to provide easy management of a storage volume without having to set up or communicate specialized storage classes across the cluster.
```

Comment:
It may be good to mention here that Kubernetes clusters on different cloud providers may already have default storageclasses installed.